exasol_integration_test_docker_environment/test/test_test_environment_certificate.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
+from exasol_integration_test_docker_environment.testing import utils
+from exasol_integration_test_docker_environment.testing.exaslct_test_environment import ExaslctTestEnvironment
+
+
+class CertificateTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print(f"SetUp {cls.__name__}")
+        # We can't use start-test-env. because it only mounts ./ and
+        # doesn't work with --build_ouput-directory
+        cls.test_environment = \
+            ExaslctTestEnvironment(
+                cls,
+                utils.INTEGRATION_TEST_DOCKER_ENVIRONMENT_DEFAULT_BIN,
+                clean_images_at_close=False)
+        cls.docker_environment_name = ""
```

Comment:
Regarding your comment to too long hostnames, this is completely fine to keep it empty to shorten the name. The max length seems to be 63 characters, because this is the max length of a label of a domain. The whole domain can't exceed 253 characters, but this includes multiple label. 
https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax,_internationalization

Reply:
But maybe add a comment, why you kept it empty.